coprocessor/fhevm-engine/fhevm-listener/Cargo.toml

```diff
@@ -18,14 +18,19 @@ alloy-rpc-types = "1.0.9"
 alloy-sol-types = "1.2.0"
 
 # workspace dependencies
+anyhow = { workspace = true }
 alloy = { workspace = true, features = ["contract", "json", "providers", "provider-ws", "pubsub", "rpc-types", "sol-types"] }
 alloy-primitives = { workspace = true }
+axum = { workspace = true }
 clap = { workspace = true }
 futures-util = { workspace = true }
+http-serde = "2.1.1"
```

Comment:
This crate doesn't seem super well-maintained, is there a way to not add it as a dependency? AFAICT, we only need it to serialize the status code, can we just convert it to an int?